core-bundle/src/Resources/config/services.yml

```diff
@@ -1023,3 +1023,7 @@ services:
             version: 4.13
             message: Using the "%service_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "Contao\CoreBundle\String\SimpleTokenParser" instead.
         public: true
+
+    Contao\CoreBundle\Monolog\SystemLogger:
+        arguments:
+            - '@logger'
```

Comment:
The logger is an optional dependency everywhere else. Should/must we follow this here as well?

Reply:
Logging may be optional for some services, but as this service is used for logging only, it relies on the logger in every method. 

You would probably make the new system logger an optional dependeny in your services, though.